k8s/wordpress/README.md

```diff
@@ -8,6 +8,22 @@ WordPress is web publishing software used to create websites and blogs.
 
 Popular open stacks on Kubernetes packaged by Google.
 
+## Design
+
+![Architecture diagram](resources/wordpress-k8s-app-architecture.png)
+
+### Solution Information
+
+WordPress solution exposes its interface on port 80.
+
```

Comment:
Consider to use inline code for port `80` and `3306` (below).

Reply:
corrected